pwchem/protocols/General/protocol_converter.py

```diff
@@ -26,15 +26,16 @@
 # **************************************************************************
 
 
-import os, shutil
+import os, shutil, mdtraj
 
 from pyworkflow.protocol.params import PointerParam, EnumParam
 from pwem.objects.data import AtomStruct
 from pwem.protocols import EMProtocol
 from pwem.convert.atom_struct import toCIF, toPdb
+from pwem.viewers import VmdView
```

Comment:
VmdView is finally not used right?

Reply:
right

Reply:
In fact, VmdView didn't work anyway because this is a protocol not a viewer. It's now removed